examples/mass_spring_damper/mass_spring_damper.ipynb

```diff
@@ -162,13 +339,14 @@
      ],
      "language": "python",
      "metadata": {},
-     "outputs": []
+     "outputs": [],
+     "prompt_number": 8
     },
     {
      "cell_type": "markdown",
      "metadata": {},
      "source": [
-      "Point $P$ can move downward in the $y$ direction and its velocity is $v$ in the downward direction."
+      "Point $P$ can move downward in the $y$ direction and its velocity is prespecified as $v$ in the downward direction."
```

Comment:
What's the difference between specified and prespecified? Can this just be "specified"?